homeassistant/components/light/mqtt.py

```diff
@@ -149,7 +156,7 @@ class MqttLight(Light):
 
     def __init__(self, name, effect_list, topic, templates, qos,
                  retain, payload, optimistic, brightness_scale,
-                 white_value_scale):
+                 white_value_scale, on_command_type='LAST'):
```

Comment:
`='LAST'` is not needed, as far as I can tell, the schema validation will take care that the default is set.

Reply:
Sure - but I don't know that some other piece of code isn't calling the constructor directly.  Having a default here keeps backward compatibility with any code (or test code)  that might not have that argument in the calling sequence.  I'm find taking it out if that's what you want...